src/settings_gui.cpp

```diff
@@ -140,6 +143,36 @@ void ShowBaseSetTextfileWindow(TextfileType file_type, const TBaseSet* baseset,
 	new BaseSetTextfileWindow<TBaseSet>(file_type, baseset, content_type);
 }
 
+std::set<int> _refresh_rates = { 30, 60, 75, 90, 100, 120, 144, 240 };
+
+/* We want 60Hz as the default for this pointer, though it gets overriden with whatever is in the config. */
+int _current_refresh_rate = 60;
+
+static const std::string HERTZ = "Hz";
+std::string _current_refresh_rate_str = std::to_string(_current_refresh_rate) + HERTZ;
+
+static inline void UpdateCurrentRefreshRate() {
```

Comment:
Apart from the fact that this will go away when you make it translatable, I meant generate the string at point of use, not make a function to update the global variable